src/block_cache/block_io.c

```diff
@@ -399,7 +508,7 @@ __wt_blkcache_write(WT_SESSION_IMPL *session, WT_ITEM *buf, WT_PAGE_BLOCK_META *
      */
     if (btree->compressor == NULL || btree->compressor->compress == NULL || compressed)
         ip = buf;
-    else if (buf->size <= btree->allocsize) {
+    else if (buf->size < 2 * WT_BLOCK_COMPRESS_SKIP /*buf->size <= btree->allocsize*/) {
```

Comment:
Is the intention to use this commented out code in the future?  If so, I think a TODO comment would be helpful, it's easy to miss.

Reply:
It needs to change for deltas, since I've never seen them large enough to bother compressing with the old heuristic. But whether this new heuristic is correct long-term is unknown. Added a comment in ab8121d7b9ccd38510fb1b627f705cf30ab03f72.